amd64/root/firmware-update.sh

```diff
@@ -36,7 +36,8 @@ typeset -A FLASHER_PARAM_VALUES=(
 # Firmware details
 # ---------------------------
 FW_PATH=/usr/share/deCONZ/firmware/
-FW_ONLINE_BASE=http://deconz.dresden-elektronik.de/deconz-firmware/
+FW_ONLINE_STABLE_BASE=http://deconz.dresden-elektronik.de/deconz-firmware/
+FW_ONLINE_BETA_BASE=http://deconz.dresden-elektronik.de/deconz-firmware/beta/
```

Comment:
Would it be an idea to not check for additional sites or paths but to allow fully qualified urls when asking for the name of the firmware?
It still should be considered how to handle md5 checks in this case.